src/anomalib/metrics/pimo/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""Per-Image Metrics."""
+
+# Original Code
+# https://github.com/jpcbertoldo/aupimo
+#
+# Modified
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from .enums import StatsOutliersPolicy, StatsRepeatedPolicy, ThresholdMethod
```

Comment:
Joao was mentioning that these policies could also be removed. Do you think we should remove them together with the validators?